### PR TITLE
New package: Logitech.GHUB version 2021.3.5164

### PR DESCRIPTION
**Pull request by alexander-danilenko**
## Logitech G HUB (v2021.3.5164)

Logitech G HUB Software lets you customize Logitech G gaming mice, keyboards, headsets, speakers, and other devices.

----

I am not related or affiliated with Logitech company and its software. Just wanted to have it in `winget` as it's one of the primary apps for me that I am installing on all my devices with Logitech G peripherals.

- The installer is publicly available.
- The installer URL is not for the latest version, but for a certain version number, which guarantees the same sha256 every time a user downloads it.

---

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/35550)

### CODE REVIEW

File: manifests/l/Logitech/GHUB/2021.3.5164/Logitech.GHUB.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: Logitech.GHUB
+Publisher: Logitech Europe S.A.
+PackageName: Logitech G HUB
+PackageVersion: 2021.3.5164
```
Comment (OfficialEsco) [inline manifests/l/Logitech/GHUB/2021.3.5164/Logitech.GHUB.yaml:4]:
```suggestion
Publisher: Logitech
PackageName: Logitech G HUB
PackageVersion: 2021.3.9205
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Huh, this does actually work now, guess they finally updated the installer

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You also need to change the version folder name 
```
from   manifests/l/Logitech/GHUB/2021.3.5164
to     manifests/l/Logitech/GHUB/2021.3.9205
```

**Comment by alexander-danilenko** [comment]
> Huh, this does actually work now, guess they finally updated the installer

Yeah, `2021.3.9205` also appears in windows software and components and app itself:

image


image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**